frontend/src/pages/acceleratorProfiles/AcceleratorProfilesRoutes.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import EditManageAcceleratorProfile from './screens/manage/EditManageAcceleratorProfile';
+import AcceleratorProfiles from './screens/list/AcceleratorProfiles';
+import ManageAcceleratorProfile from './screens/manage/ManageAcceleratorProfile';
+
+const AcceleratorProfilesRoutes: React.FC = () => (
+  <Routes>
+    <Route path="/" element={<AcceleratorProfiles />} />
+    <Route path="/manage" element={<ManageAcceleratorProfile />} />
+    <Route path="/manage/:acceleratorProfileName" element={<EditManageAcceleratorProfile />} />
+    <Route path="*" element={<Navigate to="." />} />
+  </Routes>
+);
```

Comment:
When navigating to a URL like `acceleratorProfiles?abcd`, it opens the page for `/acceleratorProfiles` but does not update the URL to reflect the change. However, when using a format like `acceleratorProfiles/abcd`, the URL is appropriately updated to `/acceleratorProfiles`. Is this behavior expected or normal?

![image](https://github.com/opendatahub-io/odh-dashboard/assets/97534722/00e0d51c-6419-465b-ab16-a7926de26df8)

Reply:
Yes, the behavior you're observing is expected and normal due to the way URLs and routing are handled on the web.

When you navigate to a URL that includes `?abcd`, the part `?abcd` is interpreted as a query string. Query strings do not change the route; they are just additional data for the server to process while loading the same route. Hence, the URL does not change from `/acceleratorProfiles` because the query string is not considered a new route, just an addition to the current route.

On the other hand, when you use the format `acceleratorProfiles/abcd`, the part `/abcd` is interpreted as a new path in the URL, which is a different route from just `/acceleratorProfiles`. If our router encounters a route it doesn't recognize, it redirects to a fallback route, which in our case is `/acceleratorProfiles`. So when you navigate to `acceleratorProfiles/abcd`, the URL is replaced with the new route because `/abcd` is considered part of the path and not just additional data like a query string.

Reply:
Great, thanks for the explanation.